.chloggen/multimod-diff-rc.yaml

```
@@ -0,0 +1,9 @@
+change_type: bug_fix
+
+component: multimod
+
+note: Changes the return code for diff to match the behaviour it is replacing
+
+issues: [404]
+
+subtext:
```

we can put the current description as subtext.

```suggestion
subtext:  This changes the exit code for `diff` to match the behaviour it is replacing in the Collector
```